config/scenario_config_coupled_ScenarioMIP.csv

```diff
@@ -1,40 +1,40 @@
-SMIPv04-VLHO-SSP2-EcBudg400-def;1;;;;;SSP2|NDC|nocc_hist|rcp2p6;y2035;raw;;;
-SMIPv04-VLLO-SSP1-PkBudg750-def;1;;;;;SDP-MC|SSP1-POP-GDP|NDC|nocc_hist|rcp1p9;y2030;raw;;;
-SMIPv04-M-SSP2-NPi2020-var;1;SMIPv04-M-SSP2-NPi2025-def;;;;;;;;;
-SMIPv04-M-SSP2-NDC_sy2030-var;1;;;;;SSP2|NDC|nocc_hist|rcp4p5;y2150;raw;;;
-SMIPv04-M-SSP2-NDC_sy2025-var;1;;;;;SSP2|NDC|nocc_hist|rcp4p5;y2150;raw;;;
-SMIPv04-L-SSP2-PkBudg1000-var_2030NPi;1;SMIPv04-L-SSP2-PkBudg1000-def;;;;;;;;;
-SMIPv04-L-SSP2-PkBudg1000-var_2030flexible;1;SMIPv04-L-SSP2-PkBudg1000-def;;;;;;;;;
-SMIPv04-H-SSP5-rollBack-var;0;;;;;SSP5|NPI|nocc_hist|rcp4p5;y2150;raw;;;
-SMIPv04-M-SSP1-NPi2025-var;0;;;;;SSP1|NPI|nocc_hist|rcp4p5;y2150;raw;;;
-SMIPv04-M-SSP1-NPi2020-var;0;;;;;SSP1|NPI|nocc_hist|rcp4p5;y2150;raw;;;
-SMIPv04-M-SSP1-NDC_sy2030-var;0;;;;;SSP1|NDC|nocc_hist|rcp4p5;y2150;raw;;;
-SMIPv04-M-SSP1-NDC_sy2025-var;0;;;;;SSP1|NDC|nocc_hist|rcp4p5;y2150;raw;;;
-SMIPv04-M-SSP2_lowEn-NPi2025-var;1;SMIPv04-M-SSP2-NPi2025-def;;;;;;;;;
-SMIPv04-M-SSP3-NPi2025-var;0;;;;;SSP3|NPI|nocc_hist|rcp4p5;y2150;raw;;;
-SMIPv04-M-SSP5-NPi2025-var;0;;;;;SSP5|NPI|nocc_hist|rcp4p5;y2150;raw;;;
-SMIPv04-L-SSP1-PkBudg1000-var;0;;;;;SSP1|NDC|nocc_hist|rcp2p6;y2035;raw;;;
-SMIPv04-VLLO-SSP2-PkBudg750-var;1;;;;;SDP-MC|SSP2-POP-GDP|NDC|nocc_hist|rcp1p9;y2030;raw;;;
-SMIPv04-VLLO-SSP2_lowEn-PkBudg750-var;1;;;;;SDP-MC|SSP2-POP-GDP|NDC|nocc_hist|rcp1p9;y2030;raw;;;
-RIKEN-VLLO-SSP2-PkBudg650-def;1;;;;;SSP2|NDC|nocc_hist|rcp1p9;y2030;raw;;;
-SMIPv04-L-SSP2-PkBudg1000-var_yr2035;1;SMIPv04-L-SSP2-PkBudg1000-def;;;;;;;;;
-SMIPv04-L-SSP2-PkBudg1000-var_yr2050;1;SMIPv04-L-SSP2-PkBudg1000-def;;;;;;;;;
-SMIPv04-VLHO-SSP2-EcBudg400-var_yr2035;1;SMIPv04-VLHO-SSP2-EcBudg400-def;;;;;;;;;
-SMIPv04-VLHO-SSP2-EcBudg400-var_yr2050;1;SMIPv04-VLHO-SSP2-EcBudg400-def;;;;;;;;;
-SMIPv04-VLLO-SSP1-PkBudg750-var_yr2035;1;SMIPv04-VLLO-SSP1-PkBudg750-def;;;;;;;;;
-SMIPv04-H-SSP3-rollBack-var_EWoff;1;SMIPv04-H-SSP3-rollBack-def;;;;;;;;;
-SMIPv04-M-SSP2-NPi2025-var_EWoff;1;SMIPv04-M-SSP2-NPi2025-def;;;;;;;;;
-SMIPv04-ML-SSP2-PkBudg2000-var_EWoff;1;SMIPv04-ML-SSP2-PkBudg2000-def;;;;;;;;;
-SMIPv04-L-SSP2-PkBudg1000-var_EWoff;1;SMIPv04-L-SSP2-PkBudg1000-def;;;;;;;;;
-SMIPv04-VLHO-SSP2-EcBudg400-var_EWoff;1;SMIPv04-VLHO-SSP2-EcBudg400-def;;;;;;;;;
-SMIPv04-VLLO-SSP1-PkBudg750-var_EWoff;1;SMIPv04-VLLO-SSP1-PkBudg750-def;;;;;;;;;
-SMIPv04-VLLO-SSP1-PkBudg750-var_CESMkUps;1;SMIPv04-VLLO-SSP1-PkBudg750-def;;;;;;;;;
-SMIPv04-VLLO-SSP1-PkBudg750-var_ProdCost;1;SMIPv04-VLLO-SSP1-PkBudg750-def;;;;;;;;;
-SMIPv04-VLLO-SSP1-PkBudg750-var_bioenergy_SustTax1p75;1;SMIPv04-VLLO-SSP1-PkBudg750-def;;;;;;;;;
-SMIPv04-VLLO-SSP1-PkBudg750-var_tradecostBio0p5;1;SMIPv04-VLLO-SSP1-PkBudg750-def;;;;;;;;;
-SMIPv04-VLLO-SSP1-PkBudg750-var_maxProdBiolc100;1;SMIPv04-VLLO-SSP1-PkBudg750-def;;;;;;;;;
+title;start;copyConfigFrom;oldrun;path_report;qos;config/scenario_config.csv;no_ghgprices_land_until;var_luc;path_gdx;path_gdx_bau;path_gdx_ref;cfg$gms$s15_elastic_demand
```

Comment:
```suggestion
title;start;copyConfigFrom;oldrun;path_report;qos;config/scenario_config.csv;no_ghgprices_land_until;var_luc;path_gdx;path_gdx_bau;path_gdx_ref;cfg_mag$gms$s15_elastic_demand
```
To be identified by the coupling scripts individual MAgPIE switches need to start with `cfg_mag`.